contracts/src/TroveManager.sol

```diff
@@ -683,9 +683,6 @@ contract TroveManager is LiquityBase, ITroveManager, ITroveEvents {
                 // Reset last zombie trove pointer if the previous one was fully redeemed now
                 lastZombieTroveId = 0;
             }
-        } else {
-            // Reset last zombie trove pointer if the previous one ended up above min debt
-            lastZombieTroveId = 0;
```

Comment:
I guess there may be an edge case, where a zombie trove, with ICR > 100% (so not skipped), gets redistribution gains and a redemption, but the total amount ends up being above `MIN_DEBT`. Then it shouldn’t be zombie anymore, but I don’t think it’s worth the extra complexity as:
- It’s an extreme edge case
- The owner can manually resuscitate the trove to get rid of the pointer.

Reply:
Good thinking! I also don't think it needs special handling, but I will make a note in the source code.

Reply:
Especially since proper resuscitation would need hints, which we lack in redemption.